pkg/sources/github/github.go

```diff
@@ -464,6 +464,8 @@ func handleRateLimit(errIn error, res *github.Response) bool {
 }
 
 func (s *Source) getReposByOrg(ctx context.Context, apiClient *github.Client, org string) ([]string, error) {
+	log := log.WithField("org", org)
```

Comment:
__optional__: I'm wondering if it would be good to attach this field on top of the existing `s.log` which looks like this: ```s.log = log.WithField("source", s.Type()).WithField("name", name)```

So like, `log := s.log.WithField("org", org)`

Or, does this make for a cleaner log message, so not worth adding the source and source name.

Reply:
Good suggestion